Adyen/UI/Form/Items/Value Pickers/Abstract/BaseFormPickerItemView.swift

```diff
@@ -8,7 +8,7 @@ import UIKit
 
 /// Represents a picker item view.
 @_spi(AdyenInternal)
-open class BaseFormPickerItemView<T: CustomStringConvertible & Equatable>:
+public final class BaseFormPickerItemView<T: CustomStringConvertible & Equatable>:
```

Comment:
What is the goal of this change? Do you have some upcoming refactors/modularization in mind?